src/web/components/LiveBlock.tsx

```diff
@@ -276,9 +277,10 @@ export const LiveBlock = ({
 				{mainElements.map((element, index) => {
 					if (typesWeStretch.includes(element._type)) {
 						return (
-							<BlockMedia>
+							<BlockMedia key={index}>
```

Comment:
As we chatted about a bit earlier, this is technically an [anti-pattern](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318) but probably isn't a massive issue. Just maybe something we need to watch out for more to set good patterns in the codebase and because live blogs might have a different reaction to changing elements in a list